static/js/topic_list.js

```diff
@@ -115,12 +116,28 @@ export function spinner_li() {
     };
 }
 
+function filter_topics_li() {
+    const render = () => render_filter_topics();
```

Comment:
I think you can just skip this as just pass `render: render_filter_topics` below; you're creating a lambda that does the same thing.

Reply:
Yeah I made this change.
(I had done this similar to function for other type of list items, should we update them too like this?)